Flint/src/main/java/flint/FlintDriver.java

```diff
@@ -1,10 +1,51 @@
 package flint;
 
 import java.util.Collection;
+import javax.tools.*;
```

Comment:
It is better to only import the specific classes you need to use, since building the project is fast and we have less dependencies.

Reply:
Should be all fixed